lib/hammer_cli_katello/repository.rb

```diff
@@ -290,6 +290,17 @@ def request_params
       build_options :without => ["uuids"]
     end
 
+    class ExportCommand < HammerCLIKatello::Command
+
+      resource :repositories, :export
```

Comment:
I am not sure why, but it doesn't work without `:repositories`. The other classes in this file have it as well.